site/content/en/docs/Guides/feature-stages.md

```diff
@@ -31,6 +31,7 @@ The current set of `alpha` and `beta` feature gates are:
 | [Player Tracking]({{< ref "/docs/Guides/player-tracking.md" >}}) | `PlayerTracking` | Disabled | `Alpha` | 1.6.0 |
 | [SDK Send GameServer on Watch execution]({{< ref "/docs/Guides/Client SDKs/_index.md#watchgameserverfunctiongameserver" >}}) | `SDKWatchSendOnExecute` | Disabled | `Alpha` | 1.7.0 |
 | Fix for RollingUpdate [Scale down](https://github.com/googleforgames/agones/issues/1625) and additional [details]({{< ref "/docs/Guides/fleet-updates.md#alpha-feature-rollingupdateonready" >}}) | `RollingUpdateOnReady` | Disabled | `Alpha` | 1.9.0 |
+| [Utilize Node ExternalDNS](https://github.com/googleforgames/agones/issues/1921) and additional [details]({{< ref "/docs/FAQ/_index.md" >}}) | `NodeExternalDNS` | Disabled | `Alpha` | 1.12.0 |
```

Comment:
Can we also wrap this in a feature shortcode please, so it won't show in current documentation when we merge it:
See: https://agones.dev/site/docs/contribute/#within-a-page

(Depending on formatting, may need to replicate the table with the additional property)

Reply:
To make this easier, just merged a PR that has this feature shortcode code in it - so if you rebase against master, you can take advantage:
(Also be good to cleanup the newly created conflicts too 😀)

https://github.com/googleforgames/agones/blame/master/site/content/en/docs/Guides/feature-stages.md#L27-L44